google/cloud/firestore_v1/query.py

```diff
@@ -359,6 +366,35 @@ def limit(self, count):
             field_filters=self._field_filters,
             orders=self._orders,
             limit=count,
+            limit_to_last=False,
+            offset=self._offset,
+            start_at=self._start_at,
+            end_at=self._end_at,
+            all_descendants=self._all_descendants,
+        )
+
+    def limit_to_last(self, count):
+        """Limit a query to return the last `count` matching results.
+
+        If the current query already has a `limit_to_last`
+        set, this will override it.
+
+        Args:
```

Comment:
Can we add the note here about mutual exclusivity with `limit` and `limit_to_last` here too?

Reply:
Done